rslib/src/deckconfig/update.rs

```diff
@@ -103,6 +103,9 @@ impl Collection {
     fn get_deck_config_use_counts(&self) -> Result<HashMap<DeckConfigId, usize>> {
         let mut counts = HashMap::new();
         for deck in self.storage.get_all_decks()? {
+            if deck.id == DeckId(1) && self.storage.deck_is_empty(deck.id)? {
```

Comment:
Your approach is perfectly valid, but you can also do

```suggestion
            if deck.id.0 == 1 && self.storage.deck_is_empty(deck.id)? {
```